core/src/main/java/io/grpc/internal/AbstractStream.java

```diff
@@ -115,7 +120,6 @@ public abstract static class TransportState
      * The default number of queued bytes for a given stream, below which
      * {@link StreamListener#onReady()} will be called.
      */
-    @VisibleForTesting
```

Comment:
The thought was that a user could use this constant to restore the default (so it'd be a user-facing constant rather than just for testing). Making the setting clearable would also suffice.

Reply:
Oh, okay. This class is in the `internal` package, so users shouldn't use it even if it is public. We would have needed to find a place to make the constant public. So, yeah, having it be clearable will be good.

Reply:
Done.